.github/workflows/check_file_size.yml

```diff
@@ -0,0 +1,18 @@
+name: Check file size
+on: push
```

Comment:
I think this means per commit push to whatever branch has a PR open?

Reply:
Updated to run on pull request